dali/kernels/imgproc/convolution/cutlass/threadblock/predicated_tile_iterator.h

```diff
@@ -117,30 +124,149 @@ namespace threadblock {
 //
 // void host(TensorView<Element, 2, layout::PitchLinear> view) {
 //
-//   using Iterator = transform::threadblock::PredicatedTileIterator;
+//   using Iterator = transform::threadblock::PositionPredicatedTileIterator;
 //
 //   typename Iterator::Params params(view.layout());
 //
 //   kernel<Iterator>(params, view.data());
 // }
+//
+//
+/// This iterator is responsible for generating matrix used for computing convolution
+/// of inner or outer axis with provided kernel window. Matrix is generated on the fly
+/// from a window stored in shared memory (should also work with global memory).
+/// Let's call this a "kernel matrix".
+///
+/// N.B. matrix used to apply convolution can be generated ahead of time by doing convolution
```

Comment:
N.B.?

Reply:
Note bene. I wanted to add a side note here.

Reply:
I think we agree with @jantonguirao not to use this abreviation.

Reply:
why?

Reply:
done :(